ce3.py

```
@@ -43,6 +44,9 @@ def __init__(self):
         # Initialize Anthropics client
         self.client = anthropic.Anthropic(api_key=Config.ANTHROPIC_API_KEY)
 
+        # Initialize Openrouter client
+        self.openrouter_client = openrouter.Client(api_key=Config.OPENROUTER_API_KEY, api_url=Config.OPENROUTER_API_URL)
```

Ruff (0.8.2)

Line too long (120 > 88)

(E501)